examples/bw_tester/bw_tester.c

```diff
@@ -21,10 +22,11 @@ int r_w_obj(struct flexalloc *fs, struct fla_pool *pool, struct fla_object *obj,
     return -ENOMEM;
   }
 
+  int rand_i_p = 1;
```

Comment:
Can you split out the changes for bw_tester into a separate commit? I know they are small but it is easier for review and for people to track  changes.

Reply:
Done.
And I actually noticed that it would be better to use an actual random number so I modified it just a bit.